engine/gamesys/src/gamesys/test/resource/create_texture.script

```diff
@@ -69,9 +69,10 @@ local tparams_2d_array = {
     format         = graphics.TEXTURE_FORMAT_RGB
 }
 
-local tparams_2d_array_missed_param = {
+local tparams_2d_array_zero_page_count = {
     width          = 128,
     height         = 128,
+    page_count     = 0,
```

Comment:
By default if parameters isn't passed it set to 1. As result `resource.create_texture` with such parameters doesn't produce assert.